src/benchq/quantum_hardware_modeling/hardware_architecture_models.py

```diff
@@ -68,10 +70,9 @@ def __init__(
         surface_code_cycle_time_in_seconds: float = 1e-3,
     ):
         self.physical_qubit_error_rate = physical_qubit_error_rate
-        # TODO: PJ check with Simon about this number
         self.surface_code_cycle_time_in_seconds = surface_code_cycle_time_in_seconds
 
-    def get_hardware_resource_estimates(self, resource_info: ResourceInfo):
+    def get_hardware_resource_estimates(self, resource_info):
```

Comment:
Can we keep the type hint for `resource_info`?

Reply:
Yes! Not sure why that got deleted.